tests/test_piiremover.py

```diff
@@ -43,12 +43,37 @@ def test_pii_remover_catches_phone_numbers(self):
         self.assertTrue(pii_remover('This is not a real telephone number (03069) 990123!') == success)
         self.assertTrue(pii_remover('This is not a real telephone number 03069 990123!') == success)
 
+    def test_pii_remover_catches_credit_card_numbers(self): 
+            
+        '''
+        Source for fake card numbers: 
+        https://www.paypalobjects.com/en_US/vhelp/paypalmanager_help/credit_card_numbers.htm
+        '''
+
+        success = 'My fake credit card number is {{CREDIT CARD}}, thank you!'
+        
+        self.assertTrue(pii_remover('My fake credit card number is 378282246310005, thank you!') == success)           
```

Comment:
Thanks for this. 

* Agreed - will switch over to assertEqual
* I've shied away from single digit redaction because the manual classifiers noted that lots of useful information like 'V5C' and 'M25' was being lost. Agree though, having a backstop for long numbers would fill this gap.
* I'm going to switch over to using scrubadub, from a fork for now - I think this is still preferable to the bespoke piiremover.py.
* Eventually redaction will be done by the admins of the tool, not me, but I agree - let's wait until I've had more experience of actually doing it.